app/src/main/java/fr/free/nrw/commons/theme/NavigationBaseActivity.java

```diff
@@ -139,6 +139,11 @@ private void setUserName() {
             drawerLayout.closeDrawer(navigationView);
             AchievementsActivity.startYourself(NavigationBaseActivity.this);
         });
+        TextView userName = navHeaderView.findViewById(R.id.username);
+        userName.setOnClickListener(v -> {
```

Comment:
I think it would make sense to somehow extract the onclick listener to a method or something, so that you do not simply need to duplicate the code, but instead call a function :)